language/use-cases/prod-catalog-enrichment/genai_prod_catalog_enrichment.ipynb

```diff
@@ -0,0 +1,2071 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    def load_json(path):
</pre><p>I don't think <code>load_json write_json</code>  need to be their own functions.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/378/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Not sure , why ? I want to keep the helper utilities abstracted from the actual code

Reply:
I would say just because they are just a couple of lines each.